morf-core/src/main/java/org/alfasoftware/morf/metadata/DataValueLookupBuilderImpl.java

```diff
@@ -361,7 +361,7 @@ private final <STORED, RETURNED> RETURNED getAndConvertByName(String columnName,
    * @return The value.
    */
   @SuppressWarnings("unchecked")
-  private final <STORED, RETURNED> RETURNED getAndConvertByIndex(Integer i, ValueMapper<STORED, RETURNED> mapper) {

```

Comment:
Not related to main PR - just flagged by the IDE,  static is redundant on a private method.